packages/experimental/Tooltip/src/Tooltip.types.ts

```diff
@@ -0,0 +1,63 @@
+import type * as React from 'react';
+
+export const tooltip = 'Tooltip';
```

Comment:
>tooltip [](http://example.com/codeflow?start=13&length=7)

we should add "Name" to this variable to make it clear it's the name of the component.